in-depth/submission_guidelines.md

```diff
@@ -0,0 +1,177 @@
+* A `thumbnail` screenshot of your item.
+  * If you omit a thumbnail image, the page will automatically use our default thumbnail and an alternate style.
+
+Here's an example of how we keep this section organized:
+
+```
+.
+├── _plugins
+│   ├── images
+│   │   ├── fullsize
+│   │   │   ├── plugin_1.png
+│   │   │   ├── plugin_1-2.png
+│   │   │   └── plugin_2.png
+│   │   └── thumbnails
+│   │       ├── plugin_1.png
+│   │       └── plugin_2.png
+│   ├── plugin_1.md
+│   └── plugin_2.md
+└── _themes
+    ├── images
+    │   ├── fullsize
+    │   │   ├── theme_1.png
+    │   │   ├── theme_1-2.png
+    │   │   └── theme_2.png
+    │   └── thumbnails
+    │       ├── theme_1.png
+    │       └── theme_2.png
+    ├── theme_1.md
+    └── theme_2.md
+```
+
+## Screenshot Specifics
+
+Each section, [Themes][gh-themes] and [Plugins][gh-plugins], has an `images` folder.  `images` is further separated into `fullsize` and `thumbnails` folders.  It's pretty self explanatory that fullsize and thumbnail images should go in their respective folders.
+
+Our recommended naming convention is to name all three of your files (the markdown and the two images) with the same name, but their respective extensions.  In the previous section's example, this would be `theme2.md`, `fullsize/theme2.png`, and `thumbnails/theme2.png`.  Any additional `fullsize` image names should start with this name, but you are free to add your own variation to distinguish between them (`theme2_blog.png`, `theme2_front_page.png`, `theme2-2.png`, etc).  This naming convention will help us and others better identify which images belong to which item.
+
+For your image format, we'd ask that you pick whichever format best optimizes your image size vs its quality.  If your image has large blank or simplistic sections, a `.png` image will provide the best quality at a reasonable file size.  If your image is busy or has a lot of fine detail, we'd recommend you using a `.jpg` with a slightly reduced quality (80-95% should provide a good balance).  If you're unfamiliar with the differences between these formats, you can experiment to determine which option will provide the best balance of quality and file size.  If you have any questions, we are [here to help][GettingHelp].
+
+For thumbnail images, you should use a resolution of `270x220`.  The thumbnails are a locked size on the website, so providing a different resolution will result in your image being squashed.
+
+Fullsize image resolution isn't as strict.  These images will be proportionally scaled to the right size.  We do however recommend a resolution of at least `1440x900`, as these images can be viewed fullscreen by clicking on their magnifying glass icon.
+
+### Specifications
+
+* Match the filename of your `.md`
+* Choose an Optimal Format, `.png` or `.jpg`
+* Fullsize: `1440x900` or larger
+* Thumbnails: `270x220`
+
+## Metadata Breakdown
+
+### Heading
```

Comment:
Not sure what to do here.  I used the headings just because they're a quick and easy way to break the content in to subsections.

The first few items aren't a big deal, but once I got down to `Available Categories` it got a little harder to format.

For example, definition lists would probably work good for this section... but it becomes a bit more complicated with the nested sections.  Nested definition lists is also somewhat of a formatting nightmare, but could work.  Also, we don't currently have styles for definition lists.

I'm going to hold off on it for the moment, but let me know what you think about it.

Is there an easier way you'd layout this section to avoid the problem?  For that matter is your problem with the appearance of the headings or the semantics of it?  Give me some ideas of what you'd like and I'll work on it.

Oh, and **bold** doesn't work.  Much too small.